server/app/lib/actions/fusor/configure_host_groups.rb

```diff
@@ -134,7 +135,8 @@ def find_or_ensure_hostgroup(deployment, product_type, organization_id, lifecycl
           # Note: when setting the arch, medium and ptable, we assume that there will only be 1
           # associated with the operating system.  If we need to support multiple in the future,
           # we'll need to update this behavior.
-          hostgroup = ::Hostgroup.create!(hostgroup_params)
+          hostgroup = ::Hostgroup.new(hostgroup_params)
+          hostgroup.save!
```

Comment:
@cfchase and I looked at this for quite awhile yesterday. I am fairly confident that create! is working in production and development mode, however when running tests create! (and create) fails to create the host and returns nil, causing the test to fail. While a little wary about changing code to satisfy tests it makes me wonder if there is an issue with using create! and this should be equivelant.